agent/util-scripts/test-bin/rpmdev-vercmp

```diff
@@ -3,18 +3,27 @@
 args="${@}"
 echo "${0} ${args}" >> ${_testlog}
 
+# We are mocking the behavior of rpmdev-vercmp where the behavior of this
+# mock is directed by the 2nd arguments name.  This allows specific test
+# cases to request a certain behavior from rpmdev-vercmp.
+
 if [[ "${2}" == "not-greater-equal-1.0.0" ]]; then
+    echo "${1} <= ${2}"
     _rc=12
 elif [[ "${2}" == "greater-equal-1.0.0" || "${2}" == "greater-1.0.0" ]]; then
+    echo "${1} >= ${2}"
     _rc=11
 elif [[ "${2}" == "not-greater-1.0.0" ]]; then
+    echo "${1} == ${2}"
     _rc=0
 elif [[ "${1}" == "${2}" ]]; then
-    # Mimic behavior of rpmdev-vercmp by exiting with 0 (for unit tests we always
-    # determine the RPM arguments are an exact match).
+    # Mimic behavior of rpmdev-vercmp by exiting with 0 (for unit tests we
+    # typically are given exact matches).
     echo "${1} == ${2}"
     _rc=0
 else
+    # If we are not given an exact match, we always report that the 1st
+    # argument is greater than the 2nd.
     echo "${1} >= ${2}"
     _rc=12
```

Comment:
I'm thinking that my grip on this stuff is less than tenuous, but, my understanding is that, if the first argument is greater than the second, we should be returning `11`, not `12`....